src/test/java/seedu/simplykitchen/logic/commands/CommandTestUtil.java

```diff
@@ -99,30 +99,31 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
      * Executes the given {@code command}, confirms that <br>
      * - a {@code CommandException} is thrown <br>
      * - the CommandException message matches {@code expectedMessage} <br>
-     * - the address book, filtered person list and selected person in {@code actualModel} remain unchanged
+     * - the SimplyKitchen inventory, filtered food list and selected food in {@code actualModel} remain unchanged
      */
     public static void assertCommandFailure(Command command, Model actualModel, String expectedMessage) {
         // we are unable to defensively copy the model for comparison later, so we can
         // only do so by copying its components.
-        AddressBook expectedAddressBook = new AddressBook(actualModel.getAddressBook());
-        List<Person> expectedFilteredList = new ArrayList<>(actualModel.getFilteredPersonList());
+        FoodInventory expectedSimplyKitchenInventory =
```

Comment:
Very minor, but should this be expectedFoodInventory instead? Line 112 and 117 as well?

Reply:
Thanks!